py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -2098,6 +2123,8 @@ def calculate_strain(
 
     # Loop over all probe positions
     for rx, ry in tqdmnd(
+        # range(220,221),
+        # range(40,41),
```

Comment:
```suggestion
```

Reply:
Good catch for both.  I tried to figure out the Lobato potential prefactors to properly set the output units, but no luck so far.